### DISCUSSION TITLE
Add disabled state (and L&F) to gauge

### DISCUSSION THREAD

**Discussion by ArdJonker** [home-assistant/frontend#12382] [Dashboard]
Sometimes a gauge is temporarily "not applicable" and then the look and feel should represent this. It seems logical to use a "disabled state" for this situation even though the gauge is not actionable (a thus can not be disabled).
Normal L&F:
<img width="301" alt="gauge" src="https://user-images.githubusercontent.com/6270792/164654327-eb6cb81d-cace-4379-a733-1d8702f03b62.png">

Disabled L&F:
<img width="301" alt="gauge" src="https://user-images.githubusercontent.com/6270792/164660367-03627f67-1436-448e-a750-9ecfcd63beb2.png">
(This is but a rough example)

**Comment by leon-v** [comment]
This would be nice. I use one for the frequency of my solar power Inverter, which at night looks like a big error.

**Comment by ildar170975** [comment]
Good idea.
Tried to use card-mod like
```
      type: gauge
      entity: ...
      card_mod:
        style: |
          {% if not has_value(config.entity) -%}
          ha-gauge {
            filter: saturate(0.1);
          }
          {%- endif %}
```
to make the svg part greyed - but the whole gauge is not shown if the entity is `unavailable`.

**Comment by phsdv** [comment]
I would like this too. I have a sensor that is not always on. When it is not on, the data the sensor returns is 'None'. When the gauge sees 'None' or other non numerical it should go to the disabled state. Now I get an error like this one: "Entity is non-numeric: sensor.c700_v2_0406_volume" instead of a Gauge (that is disabled)

**Comment by Jump-333** [comment]
I think that it could be nice if the needle just disappeared, but the entity state would still show NaN or unavailable, so I'm not sure. Making it just B&W would probably be confusing
